src/tensors/factorizations.jl

```diff
@@ -221,7 +221,8 @@ function rightnull!(t::AdjointTensorMap{S}; alg::OrthogonalFactorizationAlgorith
     return adjoint(leftnull!(adjoint(t); alg = alg'))
 end
 function LinearAlgebra.svd!(t::AdjointTensorMap{S}; trunc::TruncationScheme = NoTruncation(), p::Real = 2, alg::Union{SVD,SDD} = SDD()) where {S<:EuclideanSpace}
-    return map(adjoint, reverse(svd!(adjoint(t); trunc = trunc, p = p, alg = alg)))
+    u, s, vt, err = svd!(adjoint(t); trunc = trunc, p = p, alg = alg)
+    return map(adjoint, (vt, s, u, err))
```

Comment:
Maybe `map(adjoint, (vt, s, u))..., err`. There is no harm in taking the `adjoint` of `err`, but also no need. 

Also, while writing the documentation, I started to second guess my decision of returning the arguments as `u`, `s` and `vt`, while for Julia Base (and Matlab),  `svd` returns `u`, `s` and `v`. While typically `vt` is more useful, it does seem against the style / guidelines to overload a Julia Base function and then have it do something (slightly) different.

Reply:
Yeah, not adjointing `err` would indeed be cleaner.

I see the point in following Julia Base, even though I wish they had gone with a different convention (the NumPy one) themselves. I'm happy to accommodate my code if you decide to make the change, but it could cause some nasty silent breakage for some people, if they are SVDing matrices and `v` and `vt` have the same shape and thus you can replace one with the other without anything exploding. Then again, I don't know how much stability you even want to guarantee at this time.

Reply:
Hi Markus, I already merged your PR so don't bother. I might change this later. 

I didn't know that Python did return `vt` or, as they called (better), `vh`. That's great. I guess there are more people coming from Python than from Matlab. So maybe the right course of action here is for `TensorKit` to export `svd` as an independent method from `LinearAlgebra.svd`, and make it unnecessary for people to ever do `using LinearAlgebra` in combination with `using TensorKit`. This than matches with how  TensorKit exports `eig` and `eigh`, aside from also overloading the type-unstable method `LinearAlgebra.eigen`. The narrative is then that TensorKit pretends to be closer to Numpy than to Matlab or to Julia's `LinearAlgebra` module.